setup.py

```diff
@@ -4,6 +4,9 @@
     raise RuntimeError("setuptools >= 41 required to build")
 
 setup(
-    use_scm_version={"write_to": "src/virtualenv/version.py", "write_to_template": '__version__ = "{version}"'},
+    use_scm_version={
```

Comment:
By the way, modern setuptools-scm now advertise `pyproject.toml` as a main location for this config entries: https://github.com/pypa/setuptools_scm#pyprojecttoml-usage.
That is if you bump the deps to `"setuptools_scm[toml]>=3.4"`.

Reply:
Sadly centos do not like yet that, we had modern but reverted the change to be more graceful on centos.

Reply:
We must take seriously our support list https://virtualenv.pypa.io/en/latest/installation.html#python-and-os-compatibility